data/core.yaml

```diff
@@ -1941,6 +1941,9 @@ en:
       tag_as_disconnected:
         title: Tag as disconnected
         annotation: Tagged very close features as disconnected.
+      tag_as_not:
+        title: "Tag as <strong>not:</strong> {name}"
```

Comment:
Translators will need to know not to translate “strong” and “not” literally. I’m also not sure every language will find it convenient to use a word for “not” this way. The use of the `not:` namespace is essentially an implementation detail for those familiar with raw tags, so I think we could remove the semicolon without sacrificing clarity.

Reply:
I agree, good points on translation 👍